package.json

```diff
@@ -7,6 +7,7 @@
 	},
 	"dependencies": {
 		"@the-collab-lab/shopping-list-utils": "^2.0.0",
+		"date-fns": "^2.29.3",
```

Comment:
Yes, very good! 👏
It's a good idea to handle everything regarding date and time and, in particular, the difference between two dates or times with a library, it's very easy to make a mistake otherwise (we just had a bug at work where someone did this manually and things broke for us on March 1 as the system was expecting February to have at least 30 days...)

Reply:
🕺😅